pil-analyzer/src/condenser.rs

```diff
@@ -676,8 +677,12 @@ fn to_constraint<T: FieldElement>(
             }
             .into()
         }
-        "Lookup" | "Permutation" => {
-            assert_eq!(fields.len(), 2);
+        "Lookup" | "Permutation" | "PhantomLookup" | "PhantomPermutation" => {
```

Comment:
This converts the PIL constraint to a Rust Identity, which is essential to be able to use it.